docs/catalog/cloud/match-slo.md

```diff
@@ -0,0 +1,39 @@
+---
+version: 1.0
+submitted_by: danielmeixner
+published_date: TBD
+category: cloud
+tags: 
+ - Cloud
+ - Reliability/Availability
+ - Monitoring
+---
+
+# Match your service level objectives to business needs
+
+## Description
+
+If service downtimes are acceptable it's better to not strive for 100% uptime to design the solution according to real business needs. Lower uptime guarantees can help reduced energy consumption by using less infrastructure components.
```

Comment:
I would argue not to use _100% uptime_ as it's impossible. How about just the highest availability or the highest uptime?

Reply:
done. You are totally right that 100% uptime makes no sense. I chose this as something developers strive for even though it might not be possible. Still I removed it to avoid confusion.